etcdctl/ctlv2/ctl.go

```diff
@@ -25,15 +25,23 @@ import (
 	"github.com/urfave/cli"
 )
 
-func Start() {
+func Start(apiv string) {
 	app := cli.NewApp()
 	app.Name = "etcdctl"
 	app.Version = version.Version
 	cli.VersionPrinter = func(c *cli.Context) {
 		fmt.Fprintf(c.App.Writer, "etcdctl version: %v\n", c.App.Version)
 		fmt.Fprintln(c.App.Writer, "API version: 2")
 	}
-	app.Usage = "A simple command line client for etcd."
+	app.Usage = "A simple command line client for etcd.\n"
+
+	if apiv == "" {
+		app.Usage += "\n" +
+			"WARNING:\n" +
+			"   Environment variable ETCDCTL_API is not set; defaults to etcdctl v2.\n" +
+			"   Set environment variable ETCDCTL_API=3 to use v3 API or ETCDCTL_API=2 to use v2 API.\n"
```

Comment:
don't need newline at end of this, it's automatically added. Right now it gives:
```
NAME:
   etcdctl - A simple command line client for etcd.

WARNING:
   Environment variable ETCDCTL_API is not set; defaults to etcdctl v2.
   Set environment variable ETCDCTL_API=3 to use v3 API or ETCDCTL_API=2 to use v2 API.


USAGE:
   etcdctl [global options] command [command options] [arguments...]
```

so there's an extra newline